baselines/flwr_baselines/publications/leaf/femnist/femnist/client.py

```diff
@@ -0,0 +1,132 @@
+"""Client implementation for federated learning."""
+# pylint: disable=no-member
```

Comment:
I think so, it's what I get when I drop it:
************* Module femnist.client
femnist/client.py:36:16: E1101: Module 'torch' has no 'device' member (no-member)
femnist/client.py:109:12: E1101: Module 'torch' has no 'device' member (no-member)

Reply:
Does that happen on the CI tests or on PyCharm? Because I had no such issues and those lines on the baseline project.toml should prevent such issues:
```
[tool.pylint.typecheck] 
 generated-members="numpy.*, torch.*"
```

Reply:
I think I check it locally but without running it such that the pyproject was seen. I've just pushed the changes. I'll check the ci

Reply:
It works.